pyenvbuilder/main.py

```diff
@@ -4,12 +4,14 @@
 import argparse
 import logging
 from pyenvbuilder import __version__
+from .commands import check
 
 
 logger = logging.getLogger(__name__)
+COMMANDS = [check]
```

Comment:
If we make `COMMANDS` be a dict, we can skip the check at line 27.

Something like this:

```python
COMMANDS = {'check': check}
...

def launch(command, **kwargs):
    """
    Launch pyenvbuilder with the command specified by `command`.
    Additional parameters are passed to the command.

    Parameters
    -----------
    command : str
    """"
     ...
     log_level = kwargs.get('log_level', 'INFO')
     ...
     cmd = COMMANDS.get(command)
     if cmd is not None:
         cmd(**kwargs)
     else:
         # this is probably just the filename / directory for batch operation
```